### PR TITLE
added currentAccumulator support

### LINKED ISSUE #51: Serde `RevocationStatusList` field name
The proposed `RevocationStatusList` field name that contains the current accumulator value is [currentAccumulator](https://github.com/hyperledger/anoncreds-spec/pull/125). 

As the accumulator is in the `RevocationRegisty` struct in libursa, the `accum` field is private and serde with the same field name. 

Current situation is to keep the field name `accum`  but we will need to either update the specs or find a workaround for serde.

Hacky approach, but also create a PR (and think about backwards compatibility).

### PR DESCRIPTION
**Pull request by blu3beri**
closes #51 

Signed-off-by: blu3beri <blu3beri@proton.me>

### CODE REVIEW

File: src/data_types/rev_reg.rs [hunk 1]
```diff
@@ -35,22 +37,146 @@ pub struct RevocationRegistryDelta {
...
+            fn visit_str<E: serde::de::Error>(
+                self,
+                value: &str,
+            ) -> Result<UrsaRevocationRegistry, E> {
+                let accum = Accumulator::from_string(value).map_err(de::Error::custom)?;
+                Ok(UrsaRevocationRegistry(accum))
+            }
+
+            // fn visit_seq<V>(self, mut seq: V) -> Result<UrsaRevocationRegistry, V::Error>
+            // where
```
Comment (TimoGlastra) [inline src/data_types/rev_reg.rs:119]: Can this he removed?

**Review by TimoGlastra — APPROVED** [review]

File: src/data_types/rev_reg.rs (hunk 1 above)
Comment (blu3beri) [reply src/data_types/rev_reg.rs:119]: Yeah I created a PR, but it should be more a draft as not all the tests are working, yet. Once all the tests are working I will remove the unused code.

**Review by TimoGlastra — APPROVED** [review]
CI is failing 🙃

**Comment by blu3beri** [comment]
> CI is failing 🙃 
> 
> 
> 
> 

Yeah it's almost finished :D

**Pull request merged**